DESCRIPTION

```diff
@@ -20,7 +20,7 @@ Depends:
 Imports:
     httpcache,
     miniUI,
-    rstudioapi
+    rstudioapi(>= 0.4)
```

Comment:
Minor point, but I've always seen a space before the open parentheses in these version declarations. If check passes, then this must be valid I guess.